config/jobs/kubernetes/sig-aws/eks/k8s-aws-eks-presets.yaml

```diff
@@ -1,23 +1,16 @@
 presets:
 - env:
-  # URL to download 'kubectl', required for 'kubectl' calls to EKS (https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html)
-  # TODO: use upstream 'kubectl'
-  - name: AWS_K8S_TESTER_EKS_KUBECTL_DOWNLOAD_URL
-    value: https://amazon-eks.s3-us-west-2.amazonaws.com/1.11.5/2018-12-06/bin/linux/amd64/kubectl
-  # URL to download 'aws-iam-authenticator', required for 'kubectl' calls to EKS (https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html)
-  - name: AWS_K8S_TESTER_EKS_AWS_IAM_AUTHENTICATOR_DOWNLOAD_URL
-    value: https://amazon-eks.s3-us-west-2.amazonaws.com/1.11.5/2018-12-06/bin/linux/amd64/aws-iam-authenticator
   # AWS test account credential mounted path, required for AWS API call
   - name: AWS_SHARED_CREDENTIALS_FILE
-    value: /etc/eks-aws-credentials/eks-aws-credentials
+    value: /etc/aws-cred/credentials
   labels:
-    preset-kubernetes-e2e-aws-eks-common: "true"
+    preset-aws-credential: "aws-oss-testing"
```

Comment:
can we put a comment describing what the usage should be?

Reply:
Done. Added comment above preset definition.

Reply:
thanks, this should be a nice step towards being easier to follow the config :-)